### ISSUE TITLE
Support default language value as keys in the i18n bundle

### ISSUE THREAD

**Issue by JoeFirebaugh** [ngneat/transloco-keys-manager#26]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Maybe this is all by design and the direction of Transloco, but with ngx-translate we made the keys the default en-US value.  So our string extraction process would automatically generate the en-US bundle straight out of the code.   That reduced extra overhead of actually having to manually maintain the translation json bundle.

With transloco-keys-manager, I'm see the keys getting minified by the removal of spaces and other special characters like the following:
`  {"Processingarchive...": "Processingarchive..."}`

## Expected behavior

Is there a way to configure Transloco to work like ngx-translate in this regard?

## Minimal reproduction of the problem with instructions

The above line in our bundle was generated from this block of code in one of our template files:
```
<edge-wizard-step [label]="'Restore Options' | transloco"
                      [validatingMessage]="'Processing archive...'|transloco"
                      [validate]="simulateRestore"
                      [useFormCtrlValidation]="profile">
```

**transloco.config.js**
```
module.exports = {
  rootTranslationsPath: 'src/assets/i18n/',
  langs: ['en'],
  keysManager: {
    defaultValue: "{{key}}"
  }
};
```

## What is the motivation / use case for changing the behavior?

It's just less work on developers if they can use the english translated versions of text as the keys and default values.

## Environment

<pre><code>
Angular version: 8.2.14

Transloco Version: 2.12.4

Browser:
- [x] Chrome (desktop) version 79.0.3945.130 (MacOS)
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 13.3.0
- Platform:  Mac

Others:

</code></pre>

**Comment by shaharkazaz** [comment]
@JoeFirebaugh thanks for the report, released in v2.0.2

**Comment by JoeFirebaugh** [comment]
Thanks!
